CMakeLists.txt

```diff
@@ -235,7 +236,8 @@ ENDIF (NOT DEFINED LIB_DIR_SUFFIX)
 # set default include paths
 INCLUDE_DIRECTORIES(${PROJECT_SOURCE_DIR} ${Boost_INCLUDE_DIRS})
 
-# rpath handling
+# RPATH handling (see https://cmake.org/Wiki/CMake_RPATH_handling)
+# Note: RPATH only supported under Linux!
 SET(CMAKE_SKIP_BUILD_RPATH	FALSE)
```

Comment:
I'm pretty sure libcogutil does not need it.

But now that I think about it, the other libraries, e.g. `libatomsapce.so`, do need it, so that the path /usr/local/lib/opencog can be found, and otherwise, `ld.so` would not look there. (so its not for MacOS, but generally useful)